mod/photos.php

```diff
@@ -914,7 +914,47 @@ function photos_content(App $a)
 			'$submit' => DI::l10n()->t('Submit'),
 		]);
 
-		$usage_message = '';
+		/* This block determines which setting actually limits  upload size of images:
+		 * limit of Friendica ('maximagesize') or upload_max_filesize
+		 * and outputs the lower one.
+		*/
+		// init output var in case no number value could be retrieved.
+		$maximagesize_Mbytes = 0;
+		$sizelimitedby = '';
+		{
```

Comment:
What is this curly bracket block for?

Reply:
Couple of reasons:

1. Got the feeling this might be a little messy to be placed inline in `photos_contens` and should be moved to a function, e.g. to be reused in the Admin Site. 
2. But until I am not sure where to put it best as a Function, I used the local block for hiding the wars to the rest of the function to emphasize that they are just used for the conversion and comparison.

Reply:
But they can be removed at will, no problem at all :-)